worlds/alttp/Options.py

```diff
@@ -27,6 +28,30 @@ class Goal(Choice):
     option_hand_in = 2
 
 
+class OpenPyramid(Choice):
+    """Determines whether the hole at the top of pyramid is open.
+    Goal will open the pyramid if the goal requires you to kill Ganon, without needing to kill Agahnim 2.
+    Auto is the same as goal except if Ganon's dropdown is in another location, the hole will be closed."""
+    display_name = "Open Pyramid Hole"
+    option_no = 0
+    option_yes = 1
+    option_goal = 3
+    option_auto = 4
+    default = option_auto
```

Comment:
The website will convern "no"/"yes" to "false"/"true" in the exported yaml, so this would cause an exception. Need to add some aliases to prevent that from happening.

```suggestion
    alias_false = 0
    alias_true = 1
    default = option_auto
```